microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -289,13 +337,105 @@ def xfstesting_btrfs_nvme_datadisk_validation(
             excluded_tests=self.EXCLUDED_TESTS,
         )
 
+    @TestCaseMetadata(
+        description="""
+        This test case will run cifs xfstests testing against
+         azure file share.
+        """,
+        requirement=simple_requirement(
+            min_core_count=16,
+            supported_platform_type=[AZURE],
+        ),
+        timeout=TIME_OUT,
+        priority=3,
+    )
+    def xfstesting_azure_file_share_validation(
+        self, log: Logger, environment: Environment, log_path: Path
+    ) -> None:
+        if isinstance(environment.platform, AzurePlatform):
+            node = cast(RemoteNode, environment.nodes[0])
+            platform = environment.platform
+            information = environment.get_information()
+            resource_group_name = information["resource_group_name"]
+            location = information["location"]
+            random_str = "".join(
+                random.choices(string.ascii_lowercase + string.digits, k=10)
+            )
+            storage_account_name = f"lisa{random_str}sc"
```

Comment:
How about naming them like f"lisasc{random_str}"? So they can be sorted together.

Reply:
updated